### PR TITLE
fix flaky TestKeyHandler_DeleteKeyWithQuota

### PR DESCRIPTION
**Pull request by jeffy-mathew**
## Description
This PR fixes flaky `TestKeyHandler_DeleteKeyWithQuota`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

File: gateway/api_test.go [hunk 1]
```diff
@@ -565,11 +565,6 @@ func TestKeyHandler_DeleteKeyWithQuota(t *testing.T) {
 						}}
 					})
 
-					withAccess := CreateStandardSession()
```
Comment (jeffy-mathew) [inline gateway/api_test.go:568]: `withAccess` isn't used anywhere hence removing

**Review by titpetric — APPROVED** [review]
I was sure we deleted these "go" invokes already, i need to lie down

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4878/merge
Commit:  
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4445626680)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/release-4-lts
Commit: 6a1f2ec9ca831142fbdb9ab5bd126882e1661a9e fix flaky TestKeyHandler_DeleteKeyWithQuota (#4878)



## Description
This PR fixes flaky `TestKeyHandler_DeleteKeyWithQuota`



## Related Issue






## Motivation and Context



## How This Has Been Tested






## Screenshots (if appropriate)

## Types of changes



- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test
coverage to functionality)

## Checklist





- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning
why it's required
- [ ] I would like a code coverage CI quality gate exception and have
explained why
Triggered by: push (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4445860861)

**Pull request merged**